index.html

```diff
@@ -20,46 +20,17 @@ <h1>Leagues</h1>
 </ol>
 <p><span class="judesliga">Judesliga</span> was made by Jude Rasmussen whom which it is named. <span class="oli-league">Oli League</span> is named after Oliver Rasmussen. Obviously, <span class="andezliga">Andezliga</span> is named after Anders Rasmussen. The three leagues were made by three brothers so that's why the three leagues are one system. Three teams get relegated to <span class="oli-league">Oli League</span> and 3 go up to <span class="judesliga">Judesliga</span> from <span class="oli-league">Oli League</span>.</p>
 <h3 id="andezliga-heading">Andezliga (Year 1)</h3>
-<!--
 <pre>
-Promoted:
+Promoted from last year:
 HTF
-HTT
 UTT
+HTT
 
-Getting ready for next year shown below.
-
-       P   GD  PTS
-REL | 234 | 0 | 0 |
-UTF | 134 | 0 | 0 |
-??? | 124 | 0 | 0 |
-??? | 123 | 0 | 0 |
-
-       P   GD  PTS
-HTW | 234 | 0 | 0 |
-RHT | 134 | 0 | 0 |
-UPR | 124 | 0 | 0 |
-PTU | 123 | 0 | 0 |
-
-       P   GD  PTS
-HRA | 234 | 0 | 0 |
-URA | 134 | 0 | 0 |
-URO | 124 | 0 | 0 |
-HRO | 123 | 0 | 0 |
+REL = Relagated from Oli League
 
-       P   GD  PTS
-HAB | 234 | 0 | 0 |
-UAB | 134 | 0 | 0 |
-HPR | 124 | 0 | 0 |
-SLP | 123 | 0 | 0 |
+1OG = 1st of one of the group stages
```

Comment:
Should we call this 1OGS?

Reply:
Um... no, because you know how for example Sailopias letters are "SLP" if I put "1OGS" it would not look nice in my opinion for a reason that is really hard to explane.

Reply:
Well I think it would be easier to understand for most peple.

Reply:
Also Sailopia has more than one (so it has extra not less).

Reply:
ayayay 'o'=

Reply:
no
PIRIOD

Reply:
Okay.

Reply:
I mean .